### PR TITLE
GraphQL server side implementation

### PR DESCRIPTION
**Pull request by Hellblazer**
Generate GraphQL query schemas for facets in a workspace.

### CODE REVIEW

File: model/sql/com/chiralbehaviors/CoRE/schema/ruleform/tables/status_code_network_authorization.xml [hunk 1]
```diff
@@ -50,7 +50,7 @@
         <addForeignKeyConstraint baseTableSchemaName="ruleform " referencedTableSchemaName="ruleform"
             baseColumnNames="classifier" baseTableName="status_code_network_authorization" constraintName="status_code_network_authorization_classification_fkey"
             deferrable="false" initiallyDeferred="true" onDelete="CASCADE" onUpdate="CASCADE" referencedColumnNames="id"
-            referencedTableName="status_code" />
```
Comment (meleemistress) [inline model/sql/com/chiralbehaviors/CoRE/schema/ruleform/tables/status_code_network_authorization.xml]: The classification should be status code. Change the classifier to relationship instead. Same with all the auth files

Comment (Hellblazer) [inline model/sql/com/chiralbehaviors/CoRE/schema/ruleform/tables/status_code_network_authorization.xml]: No, it's a relationship.  The classification is status code.

Comment (meleemistress) [inline model/sql/com/chiralbehaviors/CoRE/schema/ruleform/tables/status_code_network_authorization.xml]: constraintName="status_code_network_authorization_classification_fkey"

That is the classification. The one below is the classifier

Comment (Hellblazer) [inline model/sql/com/chiralbehaviors/CoRE/schema/ruleform/tables/status_code_network_authorization.xml]: The constraint is misnamed.  Note the column reference. 

The misnaming is due to the change we made a while ago, flipping the classifier and the classification meaning.  The constraint names never got updated.  Will fix.

**Pull request merged**